doc-content/enterprise-only/fuse/installing-on-fuse-eap-proc.adoc

```diff
@@ -1,18 +1,20 @@
 [id='installing-on-fuse-eap-proc']
 = Installing {FUSE} on {EAP_LONG}
-Install {FUSE_LONG} 7.6 on {EAP} 7.2.3 to use with {PRODUCT} to deploy integrated services where required.
+Install {FUSE_LONG} {FUSE_VERSION} on {EAP} 7.2.7 to use with {PRODUCT} to deploy integrated services where required.
 
 [NOTE]
 ====
-{FUSE_LONG} 7.6 is supported on {EAP} 7.2.3, whereas {PRODUCT} is supported on {EAP} {EAP_VERSION}. To ensure a stable deployment environment, install {FUSE} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
+{FUSE_LONG} 7.7 is supported on {EAP} 7.2, whereas {PRODUCT} 7.8.x and newer is supported on Red Hat {EAP} 7.3. To ensure a stable deployment environment, install {FUSE}} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
+
+{PRODUCT} 7.8.x and newer components on {EAP} 7.2 are not officially supported but Red Hat provides guidance on a commercially reasonable basis until {FUSE} is available on {EAP} 7.3. `MarshallingFormat.JSON` is not supported on {FUSE} integration with {EAP} 7.2 using `kie-camel`. The default is JAXB.
```

Comment:
```suggestion
{PRODUCT} 7.8.x and newer components on {EAP} {EAP_VERSION} are not officially supported. However, Red Hat provides guidance on a commercially reasonable basis until {FUSE} is available on {EAP} 7.3. `MarshallingFormat.JSON` is not supported on {FUSE} integration with {EAP} {EAP_VERSION} using `kie-camel`. The default is JAXB.
```

Reply:
Will change "but" to ". However,".
Same reasoning for retaining the versions as above.